formatters/svg/svg.go

```diff
@@ -23,6 +64,9 @@ func New(options ...Option) *Formatter {
 
 // Formatter that generates SVG.
 type Formatter struct {
+	fontFamily   string
+	embeddedFont string
+	fontFormat   FontFormat
```

Comment:
I think this will need some defaults if the user doesn't use one of the font options? Probably whatever was there before?

Reply:
defaults to font-family: `Consolas, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace`